src/Models/BaseElement.php

```diff
@@ -443,9 +443,8 @@ public function getRenderTemplates()
                 break;
             }
 
-            $templates[] = $value;
-            $templates[] = 'elements/' . DataObjectPreviewController::stripNamespacing($value);
-            $templates[] = DataObjectPreviewController::stripNamespacing($value);
+            $templates[] = $value . $suffix;
+            $templates[] = DataObjectPreviewController::stripNamespacing($value) . $suffix;
```

Comment:
I left the same comment on #121 - with namespaced templates we actually _do_ want to keep the namespace rather than sanitising them, so maybe this logic can be removed now?